util/signing/hashicorp-vault/src/cryptography/ed25519/mod.rs

```diff
@@ -0,0 +1,28 @@
+use aws_sdk_kms::types::{KeySpec, KeyUsageType, SigningAlgorithmSpec};
+use signer::cryptography::secp256k1::Secp256k1;
+
+/// Defines the needed methods for providing a definition of cryptography used with AWS KMS
+pub trait AwsKmsCryptography {
```

Comment:
Why is the same trait defined both here and in the aws-kms crate, and why is it in a module named after a specific curve in each case? Do you mean to work around the orphan rule this way, rather than defining and implementing it for all curves once in the `signer` crate? This is confusing, as we don't get to see how either of the duplicate traits is supposed to be used.

Reply:
> Why is the same trait defined both here and in the aws-kms crate, and why is it in a module named after a specific curve in each case?

I think both the duplication of the trait and its placement are just mistake from copying things over too fase.